pkg/cafs/hasher.go

```diff
@@ -0,0 +1,206 @@
+package cafs
```

Comment:
this is a rename of `helpers.go` + some re-factor that git is recording as a delete and new file.

would prefer review + commit history that records re-factoring as a diff on the file ... or a single commit that contains the changes to the hashing implementation only?

Reply:
yes that is unfortunate, but that's the way it goes when a repo has accumulated too much debt

Reply:
follow-up note: open discussion thread